README.md

```diff
@@ -408,12 +411,14 @@ We have a small collection of algorithms, one for every day of the month. Scroll
     ```
     ```
     Enter Password: h3lL0
-    Time Taken to brute (approx.): 18000 sceonds
+    Time Taken to brute (approx.): 13326.48 sceonds
```

Comment:
seconds*
Also, double check the calculations just in case. Cause people will check their output with this example.

Reply:
### That's the Calculation:
- Total Number of Printable character = 92
``(92+92^2+92^3+92^4+92^5)/500000``
- Permutate from one letter to five letters until the given combination is reached.

Reply:
Ah okay got it.